windows/src/global/delphi/general/Keyman.System.Settings.pas

```diff
@@ -95,7 +95,7 @@   TKeymanSettings = class(TObjectList<TKeymanSetting>)
       ValueType: kstInteger
   );
 
-  BaseKeymanSettings: array[0..34] of TKeymanSettingBase = (
+  BaseKeymanSettings: array[0..33] of TKeymanSettingBase = (
```

Comment:
A... decrease?

Reply:
Yes, because there was one setting in the list which is not relevant -- it was never used.